Code/Editor/EditorModularViewportCameraComposer.h

```diff
@@ -29,6 +33,12 @@ namespace SandboxEditor
         SANDBOX_API AZStd::shared_ptr<AtomToolsFramework::ModularViewportCameraController> CreateModularViewportCameraController();
 
     private:
+        // AzFramework::ViewportDebugDisplayEventBus overrides ...
+        void DisplayViewport(const AzFramework::ViewportInfo& viewportInfo, AzFramework::DebugDisplayRequests& debugDisplay) override;
+
+        //! AZ::TickBus overrides ...
```

Comment:
nit: I think we are not using doxygen comments for buses overrides comments.

Reply:
Oops yes you're right about that, I can update to just `//`